vars/pushNewDockerImageTagChangePR.groovy

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/groovy
+import com.cloudbees.groovy.cps.NonCPS
+
+def call(body) {
+  // evaluate the body block, and collect configuration into the object
+  def config = [:]
+  body.resolveStrategy = Closure.DELEGATE_FIRST
+  body.delegate = config
+  body()
+
+  def flow = new io.fabric8.Fabric8Commands()
+
+  def dockerfileLocation = config.parentDockerfileLocation ?: 'Dockerfile.deploy'
```

Comment:
For anyone outside of OSIO the default is `Dockerfile` so maybe we should use that instead.